edx_proctoring/static/proctoring/js/views/proctored_exam_onboarding_view.js

```diff
@@ -25,7 +26,15 @@ edx = edx || {};
         submitted: gettext('Submitted'),
         verified: gettext('Verified'),
         rejected: gettext('Rejected'),
-        error: gettext('Error')
+        error: gettext('Error'),
+        // Enrollment modes (Note: 'verified' is both a status and enrollment mode)
+        audit: gettext('Audit'),
+        honor: gettext('Honor'),
+        professional: gettext('Professional'),
+        'no-id-professional': gettext('No ID Professional'),
+        credit: gettext('Credit'),
+        masters: gettext('Masters'),
```

Comment:
Can you change this to `Master's`? I believe that's the official spelling weuse.